app/View/Elements/menuUser.ctp

```diff
@@ -241,11 +243,19 @@
                         continue;
                       }
   
+                      // Get the plugin link array
                       $args = $menuContent['plugins'][$plugin]['coperson'][$label];
   
+                      // Always include the co_person_id and the co_id
                       $args[] = 'copersonid:' . $co['co_person_id'];
-                      $args['plugin'] = Inflector::underscore($plugin);
-  
+                      $args[] = 'co:' . $co['co_id'];
```

Comment:
Shouldn't this be `$args['co'] => $co['co_id']`? In which case the `copersonid` line above it is also wrong...

Reply:
Yes - you're correct. It was wrong in the original source, and I thoughtlessly followed suit. Fixing...

Reply:
Code style cleanup pushed (and current code rebased against the latest develop).